microscopium/_util.py

```diff
@@ -1,6 +1,82 @@
 import numpy as np
 import numbers
 
+
+def generate_spiral(dim, direction, clockwise=False):
+    """Generate an dim * dim of sequental integers in a spiral pattern.
+
+    Many HCS systems use a spiral pattern to capture multiple fields in
+    for each well. This function generates a spiral pattern of integers
+    to be used by the montaging functions.
+
+    Parameters
+    ----------
+    dim : int
+        The dimension of the spiral. The function will return a dim * dim
+        array.
```

Comment:
Yes, that's what I meant with my second comment. =D btw start coords above are [1, 1], right?